modules/rosa-hcp-aws-privatelink-create-cluster.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-aws-privatelink-creating-cluster.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-aws-privatelink-create-cluster_{context}"]
+= Creating an AWS PrivateLink cluster
+
+You can create a private cluster on {hcp-title} using the {product-title} (ROSA) CLI, `rosa`.
+
+[NOTE]
+====
+AWS PrivateLink is supported on existing VPCs only.
+====
+
+.Prerequisites
+
+* You have available AWS service quotas.
+* You have enabled the ROSA service in the AWS Console.
+* You have installed and configured the latest {product-title} (ROSA) CLI, `rosa`, on your installation host.
+* You have created a VPC with at least one private subnet.
+
+.Procedure
+
+Creating a cluster can take up to 40 minutes.
```

Comment:
in [HCP vs Classic comparison](https://file.rdu.redhat.com/eponvell/OSDOCS-8904_Privatelink-ROSA-with-HCP/rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.html#rosa-hcp-classic-comparison_rosa-hcp-sts-creating-a-cluster-quickly), we explain that provisioning time for HCP is around 10 min.